<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColormapDialog</class>
 <widget class="QDialog" name="ColormapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colormap</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Icons/colormap64.png</normaloff>:/Icons/colormap64.png</iconset>
  </property>
  <widget class="CanvasWidget" name="canvasWidget" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>390</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>488</width>
     <height>223</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Aberration</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CubeHelix1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CubeHelix2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Heat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mutant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sea</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spring</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sunbow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Velocity</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QLabel" name="twoDslider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>167</width>
        <height>146</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/Icons/tmpColormap2.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QSlider" name="redSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(200, 0, 0,128);
selection-background-color: rgba(200, 0, 0,255);
</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="greenSlider">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 200, 0,128);
selection-background-color: rgba(0, 200, 0,255);
</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="blueSlider">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 255,128);
selection-background-color: rgba(0, 0, 255,255);
</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="reverseCheckbox">
      <property name="text">
       <string>Reverse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="invertCheckbox">
      <property name="text">
       <string>Invert</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CanvasWidget</class>
   <extends>QWidget</extends>
   <header>CanvasWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
